Outcomes semester four (#71)
ALLUOS/ALLUOS.github.io: _posts/2022-03-17-outcomes.md

---
layout: post
title: Project Outcomes
sections:
  - title: Paper & Conference Talk
    tag: \#paper
  - title: IKW website
    tag: \#website
  - title: Video
    tag: \#video
  - title: Additional Resources
    tag: \#resources
  - title: References
    tag: \#references
description: Learn more about the public deliverables completed at the end of this two-year study project.
image: pic04.jpg
---

## Contents

1. [Paper & Conference Talk](#paper--conference-talk)
2. [IKW website](#website)
3. Play-through Video

> 4. Additional Resources

Author: TOC seems to be not aligned with the content. video and Additional Resources have no headline in the text.

5. [References](#references)

<div id="paper"></div>

## Paper & Conference Talk

### Conference submission

> In order to nicely wrap up all the improvements of Escapeling that were achieved throughout the second year of the project, we decided to write up these contributions as a paper. To publish the paper, we submitted a session proposal to the [Learning Ideas Conference 2022](https://www.learningideasconf.org/) - the conference where the paper written by the previous cohort of the Escapeling team was also submitted to (Johnson, Urazov and Zanoli, 2021)[^a]. More specifically, we submitted a short abstract proposing a 30-minute talk consisting of a presentation and demo elements from Escapeling. The proposal was submitted to a special track of the conference – the [ALICE special track](https://www.learningideasconf.org/alice). This is because the focus of the special track matches our application and our reported improvements particularly well, concentrating on chatbots, gamification and technology in online education. The proposal was preliminarily accepted, contingent on the acceptance of the full paper submitted on March 15th 2022. The proposal abstract can be viewed [here](https://www.learningideasconf.org/program).

Author: ```suggestion
To wrap up all the improvements of Escapeling that were achieved throughout the second year of the project, we decided to write up these contributions as a paper. To publish the paper, we submitted a session proposal to the [Learning Ideas Conference 2022](https://www.learningideasconf.org/) - the conference where the paper written by the previous cohort of the Escapeling team was also submitted to (Johnson, Urazov and Zanoli, 2021)[^a]. More specifically, we submitted a short abstract proposing a 30-minute talk consisting of a presentation and demo elements from Escapeling. The proposal was submitted to a special track of the conference – the [ALICE special track](https://www.learningideasconf.org/alice). This is because the focus of the special track matches our application and our reported improvements particularly well, concentrating on chatbots, gamification and technology in online education. The proposal was preliminarily accepted, contingent on the acceptance of the full paper submitted on March 15th, 2022. The proposal abstract can be viewed [here](https://www.learningideasconf.org/program).
```

### Framing of the paper
The goal of the paper is to present the contributions to Escapeling made since the last paper was published (Johnson, Urazov and Zanoli, 2021). Specifically, the major contributions which might be interesting to the general public comprise: different visualizations of the storyline, a new listening task, an improved storyline, the grammatical error correction (GEC) feature of the discussion task, the updated adaptive module, the new lobby platform,  technical game improvements, evaluation and testing of the respective features.  
These updates can be grouped into the conceptual classes of _user experience_ (UX) and _learning experience_ updates. Therefore, a framing focusing on novel updates of UX and the learning experience was chosen. 

> To this end, we chose to write a long (12-page) paper. In the learning experience section, we first provide general background about the app and describe existing tasks. We then first introduce the novel [listening task]({{ "" | absolute_url }}/2021/03/01/listening.html), and then present the [GEC]({{ "" | absolute_url }}/2021/02/01/GEC.html) and updated [adaptive module](TBD). We describe these two updates as contributing to a better _pedagogical feedback_ which is an important aspect of an effective learning experience (Russell and Spada, 2006)[^b]. We chose this framing because end users experience these features through feedback messages they receive when they provide their answers in the mandatory tasks (all tasks except the listening task). 

Author: 12-page long paper

> To this end, we chose to write a long (12-page) paper. In the learning experience section, we first provide general background about the app and describe existing tasks. We then first introduce the novel [listening task]({{ "" | absolute_url }}/2021/03/01/listening.html), and then present the [GEC]({{ "" | absolute_url }}/2021/02/01/GEC.html) and updated [adaptive module](TBD). We describe these two updates as contributing to a better _pedagogical feedback_ which is an important aspect of an effective learning experience (Russell and Spada, 2006)[^b]. We chose this framing because end users experience these features through feedback messages they receive when they provide their answers in the mandatory tasks (all tasks except the listening task). 

Author: ```suggestion
To this end, we write a 12-page long paper. In the learning experience section, we first provide general background about the app and describe existing tasks. We then introduce the novel [listening task]({{ "" | absolute_url }}/2021/03/01/listening.html), and afterwards present the [GEC]({{ "" | absolute_url }}/2021/02/01/GEC.html) and updated [adaptive module](TBD). We describe these two updates as contributing to better _pedagogical feedback_ which is an important aspect of an effective learning experience (Russell and Spada, 2006)[^b]. We chose this framing because the end-user experience has been improved by providing feedback messages in mandatory tasks in response to user answers (all tasks except the listening task). 
```

> In the next section, we desribe the [lobby platform](TBD), the updated [storyline]({{ "" | absolute_url }}/2021/01/01/storytelling.html) and the [visualizations]({{ "" | absolute_url }}/2021/01/02/visualizations.html). These additions are presented as improvements of the user experience, mostly due to their role in increasing the affective and motivational aspects of engagement in the game. We motivate these by literature showing the role of emotions for learning success and user experience in app design (Norman, 2005;[^c] Zallio and Damon, 2018)[^d].

Author: ```suggestion
In the next section, we describe the [lobby platform](TBD), the updated [storyline]({{ "" | absolute_url }}/2021/01/01/storytelling.html) and the [visualizations]({{ "" | absolute_url }}/2021/01/02/visualizations.html). These additions are presented as improvements to the user experience, mostly due to their role in increasing the affective and motivational aspects of engagement in the game. We motivate these by literature showing the role of emotions for learning success and user experience in app design (Norman, 2005;[^c] Zallio and Damon, 2018)[^d].
```

> We conclude the paper by presenting the [evaluation results]({{ "" | absolute_url }}/2021/04/01/evaluation.html) of the app available at the time of paper submission. These include the testing results gathered during the third semester of the project, as well as the results of three ESL student interviews and data from 14 Uni Osnabrück students who played the game and completed end user questionnaires. While the first and third evaluation strategies focus on user experience and evaluate respective improvements, the interview results provide qualitative insights into the pedagogical value of Escapeling.

Author: ```suggestion
We conclude the paper by presenting the [evaluation results]({{ "" | absolute_url }}/2021/04/01/evaluation.html) of the app available at the time of paper submission. These include the testing results gathered during the third semester of the project, as well as the results of three ESL student interviews and data from 14 Uni Osnabrück students who played the game and completed end user questionnaires. While the first and third evaluation strategies focus on user experience and evaluate respective improvements, the interview results provide qualitative insights into the pedagogical value of Escapeling, also in a classroom environment.
```

I thought it might be interesting/relevant to add the information that the ESL students gave good feedback especially for their use case in a classroom.

Maybe ESL should be written out. Also, I think 'teacher' or 'education' should be added. Otherwise, it sounds like the three education students are actually learning English themselves.
I am not even sure which acronym is the right one: https://www.eslteacheredu.org/what-is-an-esl-teacher/

Upon final acceptance and possible revisions, the paper will be published in the proceedings of the conference. 

<div id="website"></div>

## IKW Website

> <div id="references"></div>

Author: I think these divs are not needed since Jekyll transforms the ## heading to a h1 tag with a an id.<h2 id="references"
If you want to make the TOC at the beginning of the file clickable there need to be a markdown linked around the TOC list item e.g. like [References](#references)

Reviewer: That would be great, not quite sure how to do that though

Reviewer: i think i got it, but please double-check

Author: @polina-tsvilodub The general idea was right, but a headline like "Play-through Video" needs to be transformed to kebab-case like "#play-through-video"
I fixed it in another commit
